service/src/main/resources/application.conf

```diff
@@ -180,7 +180,7 @@ SunbirdMWSystem {
         "/RequestRouter/*/UserProfileReadActor"
           {
             router = smallest-mailbox-pool
-            nr-of-instances = 5
+            nr-of-instances = 25
```

Comment:
here total number of actor instance will be 25*5 =125 that seems too high , 
keep             nr-of-instances = 10

Reply:
@manzarul we are experimenting with this API since our CPU consumption in any of our APIs is not more than 15% and memory is always between 20 to 25%, so I think we should give a try once with this change for better utilization of CPU and ofCourse throughput. if this is wrong we can anyways revert back to nr of instances to 10

Reply:
ok @anmol2302